test/tools/libtesteth/Options.cpp

```diff
@@ -278,7 +277,7 @@ Options::Options(int argc, char** argv)
 	}
 	else
 	{
-		if (randomTestSeed != 0)
+		if (randomTestSeed.is_initialized())
```

Comment:
It can be `if (randomTestSeed)`.

Reply:
m_vrs in transaction signature also using is_initialized. 
I just want it to be clear that it is not the int value being checked but boost::optional